### PR TITLE
Bazel-update description and args

### PR DESCRIPTION
**Pull request by ly0708**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Updated the description in bazel.ts making it more clear.
Changed using args instead of insertValue. 

**What is the current behavior? (You can also link to an open issue here)**

The description is not clear. 

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

File: dev/bazel.ts [hunk 1]
```diff
@@ -273,7 +293,7 @@ const completionSpec: Fig.Spec = {
       name: "--nounlimit_coredumps",
       exclusiveOn: ["--unlimit_coredumps"],
       description:
-        "Raises the soft coredump limit to the hard limit to make coredumps of the server (including the JVM) and the client possible under common conditions",
+        "Don't raises the soft coredump limit to the hard limit to make coredumps of the server (including the JVM) and the client possible under common conditions",
```
Comment (fedeci) [inline dev/bazel.ts:296]: typo:
```suggestion
        "Don't raise the soft coredump limit to the hard limit to make coredumps of the server (including the JVM) and the client possible under common conditions",
```

File: dev/bazel.ts [hunk 2]
```diff
@@ -239,8 +256,10 @@ const completionSpec: Fig.Spec = {
     },
     {
       name: "--server_jvm_out",
-      insertValue: "--server_jvm_out={cursor}",
       description: "The location to write the server's JVM's output",
+      args: {
+        name: "Path",
```
Comment (fedeci) [inline dev/bazel.ts:261]: Does it accept a `filepaths` template?

Comment (ly0708) [reply dev/bazel.ts:261]: I am sure here is the doc - 
`--server_jvm_out=<path> default: see description
The location to write the server's JVM's output. If unset then defaults to a location in output_base.
Tags: affects_outputs, loses_incremental_state`

Comment (ly0708) [reply dev/bazel.ts:261]: What do you mean by filepaths?

Comment (fedeci) [reply dev/bazel.ts:261]: Some path on the machine on which the command is run onto.

Comment (fedeci) [reply dev/bazel.ts:261]: Like `~/Desktop/logs/out.log`

Comment (ly0708) [reply dev/bazel.ts:261]: I see. then I think it does accept "filepaths template".

**Review by fedeci — APPROVED** [review]
Thanks!

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot for those amazing new descriptions!

**Pull request merged**